src/it/scala/com/ing/wbaa/airlock/proxy/AirlockS3ProxyItTest.scala

```diff
@@ -7,10 +7,10 @@ import akka.stream.ActorMaterializer
 import com.amazonaws.auth.BasicSessionCredentials
 import com.amazonaws.services.securitytoken.AWSSecurityTokenService
 import com.amazonaws.services.securitytoken.model.GetSessionTokenRequest
-import com.ing.wbaa.airlock.proxy.config.{AtlasSettings, HttpSettings, StorageS3Settings, StsSettings}
+import com.ing.wbaa.airlock.proxy.config._
```

Comment:
What's everyone's opinion on using wildcards for imports? In past I avoided it (in the Java world) mostly because there could be the possibility of name clashes when importing classes.

Reply:
I think it is "auto" feature of intellij after you include more than x from particular package. Other thing is that it will not compile on name clash

Reply:
It is commonly disabled in the large opensource project. IntelliJ is quite annoying in this respect